unittests/StatusTest.php

```diff
@@ -4,6 +4,7 @@
 require_once 'class/metrolink.class.php';
 require_once 'class/status.class.php';
 require_once 'class/promises/websitehandler.interface.php';
+require_once 'unittests/dataproviders/status.dataprovider.php';
```

Comment:
I'd adopt PSR-2 (or PSR-4) for class-Naming (and autoloading). As the class is named `Dataprovider\Status` the filename would then be `Dataprovider/Status.php`.